src/main/java/com/opentable/db/postgres/embedded/EmbeddedPostgreSQL.java

```diff
@@ -365,6 +364,7 @@ public static EmbeddedPostgreSQL start() throws IOException
         private File dataDirectory;
         private final Map<String, String> config = Maps.newHashMap();
         private boolean cleanDataDirectory = true;
+        private int port = 0;
```

Comment:
Yep probably worth fixing. `dataDirectory` and `cleanDataDirectory` are also like this. I'll change them up at the same time.